yarn-project/noir-contracts/src/contracts/private_token_airdrop_contract/src/interface.nr

```diff
@@ -0,0 +1,115 @@
+/* Autogenerated file, do not edit! */
```

Comment:
Is this contract actually imported by any of the other contracts?

Reply:
It's used by the same contract to call its own functions!